### PR TITLE
Adding ZSA Orchard Action Description encoding details

### PR DESCRIPTION
**Pull request by vivek-arte**
This PR adds details of the changes to the Orchard Action encodings and the changes to the transaction format that occur due to the ZSA Protocol. It also improves the formatting of existing tables for Asset Burns.

### CODE REVIEW

File: zip-0226.rst [hunk 1]
```diff
@@ -302,30 +302,90 @@ Burn Mechanism Consensus Rules
...
++-------+---------------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Bytes | Name          | Data Type                   | Description                                                                                                               |
++=======+===============+=============================+===========================================================================================================================+
+| 32    | ``AssetBase`` | ``byte[32]``                | For the Orchard-based ZSA protocol, this is the encoding of the Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}`. |
++-------+---------------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| 8     | ``valueBurn`` | :math:`\{1 .. 2^{64} - 1\}` | The amount of the Asset Base being burnt.                                                                                 |
```
Comment (PaulLaux) [inline zip-0226.rst:387]: `The amount being burnt.` will be enough

File: zip-0226.rst [hunk 2]
```diff
@@ -302,30 +302,90 @@ Burn Mechanism Consensus Rules
...
+|                             |                          |                                           |                                                                     |
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``852 * nActionsOrchard``    |``vActionsOrchard``       |``ZSAOrchardAction[nActionsOrchard]``      |A sequence of ZSA Orchard Action descriptions, encoded per           |
+|                             |                          |                                           |the `ZSA Orchard Action Description Encoding`_. **[UPDATED FOR ZSA]**|
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``1``                        |``flagsOrchard``          |``byte``                                   |                                                                     |
```
Comment (PaulLaux) [inline zip-0226.rst:323]: We will need a spec for `flagsOrchard` and how it is translates to `finilized`. (a different pr)

**Review by PaulLaux — APPROVED** [review]
Approved with one pending comment.

File: zip-0226.rst (hunk 1 above)
Comment (vivek-arte) [reply zip-0226.rst:387]: Done.

**Pull request merged**